rows.go

```diff
@@ -20,9 +20,10 @@ type rowSet struct {
 	columns    []*tcliservice.TColumnDesc
 	columnStrs []string
 
-	offset    int
+        offset    int
```

Comment:
It seems that the indentation of this line doesn't match the one of the next line?

Reply:
thanks for the careful reminder, fixed already.